ouroboros-consensus-cardano/app/DBAnalyser/Parsers.hs

```diff
@@ -129,24 +129,35 @@ storeLedgerParser = do
     (  long "store-ledger"
     <> metavar "SLOT_NUMBER"
     <> help "Store ledger state at specific slot number" )
-  ledgerValidation <- flag LedgerReapply LedgerApply
-    (  long "full-ledger-validation"
-    <> help (  "Use full block application while applying blocks to ledger states, "
-            <> "also validating signatures and scripts. "
-            <> "This is much slower than block reapplication (the default)."
-            )
-    )
+  opts <- pApplyOpts
   doChecksum <- flag DoDiskSnapshotChecksum NoDoDiskSnapshotChecksum
     (mconcat [ long "no-snapshot-checksum-on-write"
              , help (unlines [ "Don't calculate the checksum and"
                              , "write the '.checksum' file"
                              , "when taking a ledger snapshot"
                              ])
              ])
-  pure $ StoreLedgerStateAt slot ledgerValidation doChecksum
+  pure $ StoreLedgerStateAt slot opts doChecksum
+
+pApplyOpts :: Parser ApplyOpts
```

Comment:
If `db-analyser` is in fact the only intended use case of `ValidationOff`, then I would strongly prefer to not define `ValidationOpts` in the `ouroboros-consensus` interface and rather isolate that flag within `db-analyser`.

Reply:
Validation options are gone